hack/setup-test-cluster.sh

```diff
@@ -8,10 +8,16 @@ export OC_TOOL="${OC_TOOL:-oc}"
 # Label 2 worker nodes as worker-cnf
 echo "[INFO]: Labeling 2 worker nodes with worker-cnf"
 nodes=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/worker' \
-  --selector='!node-role.kubernetes.io/master' -o name | sed -n 1,2p)
+  --selector='!node-role.kubernetes.io/master' -o name | sed -n 1,3p)
+count=1;  
 for node in $nodes
 do
-    ${OC_TOOL} label $node node-role.kubernetes.io/worker-cnf=""
+    if [ count > 2 ]; then
+        ${OC_TOOL} label $node node-role.kubernetes.io/worker-cnf-no-rt=""
```

Comment:
Out of the 3 nodes, 2 nodes will be with 'worker-cnf-no-rt' label. Is that what we want ?